core/commands/add.go

```diff
@@ -44,6 +45,7 @@ remains to be implemented.
 	Options: []cmds.Option{
 		cmds.OptionRecursivePath, // a builtin option that allows recursive paths (-r, --recursive)
 		cmds.BoolOption(quietOptionName, "q", "Write minimal output"),
+		cmds.BoolOption(silentOptionName, "x", "Write no output"),
```

Comment:
i did `qq` first, but apparently it doesnt work for our commands lib. it parses it as just `q` :(